package.json

```
@@ -37,7 +37,7 @@
   },
   "dependencies": {
     "@opam/ctypes": "0.15.1",
-    "@opam/dune": ">=1.6.0",
+    "@opam/dune": "2.5.0",
```

```suggestion
    "@opam/dune": ">=1.6.0 <=2.5.0",
```
or maybe we expose the tips to "fix" failure because it is safe here. (not for esy-sdl2) https://github.com/revery-ui/revery/issues/866
because we might forget to use the range version when releasing and then we will have 0.3.2+ with very strange deps constraints: Can we try "resolutions" fields ?

---

"expose the tips"?

I agree it would be better to sue resolutions though. Fixed in https://github.com/revery-ui/revery/pull/887/commits/e89373a3393f56f16501e37d8c238f9896223972. This should be patched upstream relatively soon anyway.

---

The tips is in the issue I have linked.

---

Honestly I wouldn't apply such a hack to my own system, let alone recommend it to others. I think recommending this resolution if people have problems with their own projects should be fine though?